Added Popeyes nutrition information.

cweirup opened the pull request:
Adding nutrition information for Popeyes USA.

Popeyes is inconsistent in terms of portions sizes. Some are weight, some fluid ounces, some in counts, some in relative sizes.

If it's easier for you, I can add Servicing Size Descriptions for any items that have a count. I did convert to grams or milliliters if they provided weights or volumes.

I've also included in the sources directory:
- PDF with nutrition info (source below)
- Excel spreadsheet I extracted from the PDF with some additional columns and formulas to derive the CSV file

Source (updated February 2023): https://plk-use1-prod.sites.rbictg.com/nutrition/PLK_Nutrition.pdf

ryanashcraft commented on popeyes.csv:
```diff
@@ -0,0 +1,61 @@
+Blackened Ranch,28.3495,,120,110,12,2,0,10,250,2,0,1,<1
+Buttermilk Ranch,28.3495,,140,140,15,3,0,10,230,2,0,1,0
+Mardi Gras Mustard,28.3495,,100,80,8,1,0,10,240,5,1,4,1
+Sweet Heat®,38.27186,,70,0,0,0,0,0,290,19,0,16,0
```
Please remove ®

ryanashcraft commented on popeyes.csv:
```diff
@@ -0,0 +1,61 @@
+﻿Name,Serving Size (g),Serving Size (mL),Calories,Calories from Fat,Total Fat (g),Saturated Fat (g),Trans Fat (g),Cholesterol (mg),Sodium (mg),Total Carb (g),Dietary Fiber (g),Total Sugar (g),Protein (g)
```
- Please remove `Calories From Fat` column
- Please rename `Dietary Fiber (g)` to `Fiber (g)`
- Please rename `Total Carb (g)` to `Carbohydrates (g)`
- Please rename `Total Sugar (g)` to `Sugars (g)`

ryanashcraft commented on five-guys.csv:
```diff
@@ -0,0 +1,40 @@
+﻿Name,Serving Size (g),Calories,Calories from Fat,Total Fat (g),Saturated Fat (g),Trans Fat (g),Cholesterol (mg),Sodium (mg),Carbohydrates (g),Fiber (g),Sugars (g),Protein (g)
```
Please remove the `Calories from Fat` column

ryanashcraft commented on five-guys.csv:
```diff
@@ -0,0 +1,40 @@
+"Fries, Little",227,526,204,23,4,0,0,531,72,8,2,8
+"Fries, Regular",411,953,370,41,7,1,0,962,131,15,4,15
+"Fries, Large",567,1314,511,57,10,1,0,1327,181,21,6,20
+A.1.Original Steak Sauce,17,15,0,0,0,0,0,280,3,0,2,0
```
Please add a space between "A.1." and "Original"

ryanashcraft commented on five-guys.csv:
```diff
@@ -0,0 +1,40 @@
+Peanut Butter Milkshake Mix-In,14,86,67,7,1,0,0,71,3,<1,1,3
+Salted Caramel Milkshake Mix-In,14,43,2,0,0,0,1,24,10,0,8,0
+Strawberry Milkshake Mix-In,14,20,0,0,0,0,0,5,0,0,5,0
+Bulk Peanuts (1 oz.),30,160,,,,,,,,,,
```
Let's remove these since they only have calories and no macro information. Also, it's very weird that they list 2 eggs as 4g.

ryanashcraft reviewed (commented):
Thanks so much for contributing these additions! In the future, might be easiest to make separate PRs for multiple restaurants.

I have a few requests for you before I merge this, if you don't mind fixing those up. Thanks!

cweirup commented:
Ryan,

Ah, my apologies. For the Popeyes one, it looks like I added a different version than my working copy. And I didn't think that my commit for Five Guys would update the PR (I'm not seasoned with working on projects that are not just me). I think I've made all of the updates you requested.

One clarification, on the Five Guys one, I removed the last three items, since as you mentioned the data is very light and potentially incorrect.

For reference on Five Guys, here is the PDF source online: https://www.fiveguys.com/-/media/public-site/files/allergen-ingredients-and-nutrition-info/allergen-guide/nutrition-allergen-march-2018-us.ashx

Again, my apologies for the issues.

ryanashcraft commented:
@cweirup No worries at all. Thanks so much for taking the time to submit these and make those changes. Will upload these to the FoodNoms DB today!
